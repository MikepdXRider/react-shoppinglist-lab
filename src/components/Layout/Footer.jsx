import React from 'react'

export default function Footer() {
    return (
        <footer>
            <h1>This is the footer component!</h1>
        </footer>
    )
}
