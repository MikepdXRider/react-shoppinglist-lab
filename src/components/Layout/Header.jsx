import React from 'react'

export default function Footer() {
    return (
        <header>
            <h6>This is the header component!</h6>
        </header>
    )
}