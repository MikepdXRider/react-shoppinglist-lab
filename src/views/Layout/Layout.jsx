import React from 'react';
import Header from '../../components/Layout/Header.jsx';
import Footer from '../../components/Layout/Footer.jsx';

export default function Layout({children}) {
    return (
        <div>
            <Header />
            {children}
            <Footer />
        </div>
    )
}
