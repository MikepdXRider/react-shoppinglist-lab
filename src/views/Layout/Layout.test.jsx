import Layout from "./Layout.jsx"
import { render } from '@testing-library/react'


it('renders layout view component', () => {
    const {container} = render(
        <Layout />
    )

    expect(container).toMatchSnapshot();
})