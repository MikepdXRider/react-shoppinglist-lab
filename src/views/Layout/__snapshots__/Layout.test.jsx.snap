// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders layout view component 1`] = `
<div>
  <div>
    <header>
      <p>
        This is the header component!
      </p>
    </header>
    <footer>
      <p>
        This is the footer component!
      </p>
    </footer>
  </div>
</div>
`;
